docs/03 - Implementing the Anchor Server/Communication/Events Schema.yml

```diff
@@ -102,92 +103,46 @@ components:
-            The status of the transaction according with the latest Platform update. Currently, we're supporting all 
-            [SEP-31 states](https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0031.md#get-transaction), exept for `pending_transaction_info_update`.
+            The status of the transaction according with the latest Platform update. Currently, we're supporting
+            [SEP-24 states](https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0024.md#fields-for-withdraw-transactions).
+            [SEP-31 states](https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0031.md#get-transaction).
           enum: [
-            "pending_sender",
-            "pending_stellar",
-            "pending_customer_info_update",
-            "pending_receiver",
-            "pending_external",
-            "completed",
-            "expired",
-            "error"
+            # Shared
+            incomplete,
+            completed,
+            refunded,
+            expired,
+            error,
+            pending_stellar,
+            pending_external,
+            # SEP24
+            pending_user_transfer_start,
+            pending_user_transfer_complete,
+            pending_anchor,
+            pending_trust,
+            pending_user,
+            no_market,
+            too_small,
+            too_large,
+            # SEP31
+            pending_sender,
+            pending_receiver,
+            pending_transaction_info_update,
+            pending_customer_info_update
           ]
-        status_change:
-          $ref: '#/components/schemas/StatusChange'
-        amount_expected:
-          $ref: '#/components/schemas/Amount'
-        amount_in:
-          $ref: '#/components/schemas/Amount'
-        amount_out:
-          $ref: '#/components/schemas/Amount'
-        amount_fee:
-          $ref: '#/components/schemas/Amount'
-        quote_id:
-          type: string
-          description: "The id of the [SEP-38](https://stellar.org/protocol/sep-38) firm quote used for the transaction."
-        started_at:
-          type: string
-          format: date-time
-        updated_at:
-          type: string
-          format: date-time
-        completed_at:
-          type: string
-          format: date-time
-          description: "When the transaction was marked as `completed` or when it was fully refunded."
-        transfer_received_at:
-          type: string
-          format: date-time
-          description: "When the Stellar transaction related to this object was received."
         message:
           type: string
           description: "This field may contain any clarifying message related to this TransactionEvent or to the transaction itself."
-        refunds:
-          $ref: '#/components/schemas/Refunds'
-        stellar_transactions:
-          type: array
-          items:
-            $ref: '#/components/schemas/StellarTransaction'
```

Comment:
It'd be good to return some Ledger identification, like the transaction or operation id, so the Anchor can reach out to the Ledger for more information.